data/strings/types_strings.txt

```diff
@@ -22084,8 +21835,7 @@
     zh-Hant = 按摩館
 
   [type.shop.mobile_phone]
-    en = Cell Phones
-    en-GB = Mobile Phones
+    en = Mobile Phone Shop
```

Comment:
It's Cell Phone Store in the US, but Mobile is not really wrong.

Reply:
fixed